modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -204,6 +204,19 @@ config TFM_IMAGE_VERSION_NS
 	  versions of both the secure firmware and non-secure firmware. This sets
```

Comment:
So the version numbers are not used anymore for rollback protection?

Reply:
Correct, not that the existing setup worked once you went past `0.0.1024` anyway.

Reply:
So the descriptions of the version number Kconfig options should be updated?